documentation/modules/proc-iot-api.adoc

```diff
@@ -13,17 +18,22 @@ endif::[]
 
 = Sending and receiving {api} data
 
-.Procedure
+== Starting the {api} consumer
 
-== Starting {api} consumer
+.Prerequisites
```

Comment:
Just a minor suggestion for prereqs: I typically state them as present conditions (so, "The IoT services are installed", "An IoT project is created", etc.)

However, if the other prereqs in this doc are phrased a certain way, I'd just match it here for consistency.